src/pages/test-report/cat-a-mod1/components/activity-code-4-modal/activity-code-4-modal.constants.ts

```diff
@@ -0,0 +1,4 @@
+export enum ModalReason {
+  SPEED_REQUIREMENTS = 'Speed requirements not met',
+  EMERGENCY_STOP = 'Serious fault recorded on emergency stop',
```

Comment:
There is going to be another reason here:

1.  SPEED_REQUIREMENTS = 'Speed requirements not met'
2. EMERGENCY_STOP_SERIOUS = 'Serious fault recorded on emergency stop'
3. EMERGENCY_STOP_DANGEROUS = 'Dangerous fault recorded on emergency stop'

Reply:
Added in